### PR TITLE
Terminal fi patch 1

### PR DESCRIPTION
**Pull request by TerminalFi**

### CODE REVIEW

File: sublime-package.json [hunk 1]
```diff
@@ -109,9 +104,9 @@
                       "markdownDescription": "linkTarget controls where documentation links go.\nIt might be one of:\n\n* `\"godoc.org\"`\n* `\"pkg.go.dev\"`\n\nIf company chooses to use its own `godoc.org`, its address can be used as well.\n\nModules matching the GOPRIVATE environment variable will not have\ndocumentation links in hover.\n"
                     },
                     "gopls.linksInHover": {
-                      "type": "boolean",
+                      "type": "any",
```
Comment (rchl) [inline sublime-package.json:107]: doesn't seem valid

File: sublime-package.json [hunk 2]
```diff
@@ -476,36 +486,14 @@
                       "markdownDescription": "(experimental) staticcheck enables additional analyses from staticcheck.io.\nThese analyses are documented on\n[Staticcheck's website](https://staticcheck.io/docs/checks/).\n"
                     },
                     "gopls.annotations": {
-                      "type": "object",
+                      "type": "map[enum]bool",
```
Comment (rchl) [inline sublime-package.json:489]: doesn't seem valid

File: scripts/update-schema-settings.py [hunk 3]
```diff
@@ -139,7 +140,9 @@ def generate_schema(self):
         raw_settings = raw_schema["Options"]["User"]
         for _, value in enumerate(raw_settings):
             current_key = f"gopls.{value['Name']}"
-            current_type = TYPE_MAP[value["Type"]]
+            print(current_key)
+            print(value["Type"])
```
Comment (rchl) [inline scripts/update-schema-settings.py:144]: should those be removed?

File: sublime-package.json (hunk 1 above)
Comment (TerminalFi) [reply sublime-package.json:107]: They now have true, false, and a string value

Comment (rchl) [reply sublime-package.json:107]: What I mean is that per JSON schema the `any` value is not valid for `type`. You can assing an array `["boolean", "string"]`.

**Comment by TerminalFi** [comment]
add to vulchrck cleanup

**Pull request merged**